tests/StorageBackendTest.php

```diff
@@ -82,16 +42,48 @@ public function testStorageAssignBackend(string $backendName): void
             'url' => getenv('KBC_MANAGE_API_URL'),
             'token' => $token['token'],
         ]);
-        $bucketId = $sapiClient->createBucket('test', 'in');
-        $bucket = $sapiClient->getBucket($bucketId);
-        $this->assertEquals($backendName, $bucket['backend']);
 
```

Comment:
Tady by IMHO měl být ještě test, že to failne na prázdném existujícím projektu a na existujícím projektu s bucketem.